docs/developers_guide.md

```diff
@@ -219,3 +219,88 @@ We plan to keep our code loosely up to date with the upstream Firecracker reposi
 to speed up testing environment setup requiring Kubernetes.
 
 * Current [eStargz](https://github.com/containerd/stargz-snapshotter) version is 0.13.0.
+
+
+## Deploying Knative functions that require GPU Support
+
+Knative functions can use GPU although only `stock-only` mode is supported. 
+
+### Install Stock Kubernetes and Knative
+
+Follow the guide to [setup stock knative](#testing-stock-knative-setup-or-images). 
+
+``` bash
+./scripts/cloudlab/setup_node.sh stock-only
+```
+
+### Install NVIDIA Driver and NVIDIA Container Toolkit 
+The script will install NVIDIA CUDA Driver and assume there’s no NVIDIA driver currently running. You can use the script provided if the install of containerd is using our script or manually edit the containerd settings following [NVIDIA's official document](https://docs.nvidia.com/datacenter/cloud-native/kubernetes/install-k8s.html#install-nvidia-container-toolkit-nvidia-docker2).
```

Comment:
please specify the specs of the platform, i.e., CUDA and GPU version, on which you tested the following commands. Give recommendations on which platforms/GPUs the commands should work. 

split the long line into several lines for finer-grain version control